.github/workflows/fatimage.yml

```diff
@@ -2,6 +2,15 @@
 name: Build fat image
 'on':
   workflow_dispatch:
+    inputs:
+      ci_cloud:
+        description: 'Select the CI_CLOUD'
+        required: true
+        type: choice
+        options:
```

Comment:
can we add a [default](https://docs.github.com/en/actions/writing-workflows/choosing-when-your-workflow-runs/events-that-trigger-workflows#providing-inputs) like this:

`default: ${{ vars.CI_CLOUD }}`

so that it defaults to whatever is set on the repo? I don't know if that will work but that would make this much more usable.